components/campaigns/AnnouncementHero.tsx

```diff
@@ -45,7 +45,7 @@ export default function AnnouncementHero({ className = '', small = false }: IAnn
   }, [activeIndex]);
 
   return (
-    <Container as='section' padding='' className={'text-center'}>
+    <Container as='section' padding='' className={`text-center ${numberOfVisibleBanners === 0 ? 'hidden' : ''}`}>
```

Comment:
Instead of adding a check over className, we can have this condition above and return `''` from the component.

Reply:
Have made the desired changed!